### PR TITLE
Add ClonalFrameML: Efficient Inference of Recombination in Whole Bacterial Genomes

### PR DESCRIPTION
**Pull request by thanhleviet**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/clonalframeml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package:
+  name: clonalframeml
+  version: "1.11"
+
+build:
+  number: 3
```
Comment (bgruening) [inline recipes/clonalframeml/meta.yaml]: This should start with 0.

File: recipes/clonalframeml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  run:
+  - libgcc # [linux]
+
+test:
+  commands:
+    - ClonalFrameML 2>&1 | grep "ClonalFrameML"
+
+about:
+  home: https://github.com/xavierdidelot/ClonalFrameML
+  license: GPLv3
```
Comment (bgruening) [inline recipes/clonalframeml/meta.yaml]: Can you provide a `license_file`

File: recipes/clonalframeml/1.11/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package:
+  name: clonalframeml
+  version: "1.11"
+
+build:
+  number: 3
```
Comment (bgruening) [inline recipes/clonalframeml/1.11/meta.yaml]: start with 0

File: recipes/clonalframeml/1.11/build.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+export CFLAGS="-I$PREFIX/include"
```
Comment (bgruening) [inline recipes/clonalframeml/1.11/build.sh]: Are these exports needed? You do not depend on any third party library as it seems.

File: recipes/clonalframeml/meta.yaml (hunk 2 above)
Comment (thanhleviet) [reply recipes/clonalframeml/meta.yaml]: Thanks for your comments. The authors did not provide the license file but mentioning that the source code follows GPLv3, should I include this one https://www.gnu.org/copyleft/gpl.html ?
Do I have to create new PR?

Comment (bgruening) [reply recipes/clonalframeml/meta.yaml]: I will discuss a general approach in an issue as including in every recipe the GPL (as hardcopy) also seems wired, we should do better.

**Pull request merged**